Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM mcr.microsoft.com/playwright:v1.22.0-focal
+FROM mcr.microsoft.com/playwright:v1.25.0-focal
```

Comment:
@christianvuerings What is this for?

Reply:
@AlbertCarreras to upgrade playwright to the latest version. In this case, we use https://hub.docker.com/_/microsoft-playwright as a base image